AppiumForMac/Server/Controller/AfMElementLocator.h

```diff
@@ -36,6 +36,6 @@ typedef enum appiumMacLocatoryStrategy
 
 -(PFUIElement*) findUsingBaseUIElement:(PFUIElement*)baseUIElement statusCode:(int*)statusCode;
 
--(void) findAllUsingBaseUIElement:(PFUIElement*)baseUIElement results:(NSMutableArray*)results statusCode:(int*)statusCode;
+-(NSArray*) findAllUsingBaseUIElement:(PFUIElement*)baseUIElement results:(NSMutableArray*)results statusCode:(int*)statusCode;
```

Comment:
nullable NSArray*

Reply:
So if we change the return to empty array then we don't need to use nullable NSArray right?

Reply:
correct